crates/common/src/compile.rs

```diff
@@ -398,10 +398,16 @@ pub fn compile_target_with_filter(
     }
 }
 
+/// Id to map from the bytecode to the source file
+pub type FileId = u32;
```

Comment:
please no public type aliases for primtive values. either make it private or add a new wrapper type.

and move to top of file

Reply:
I think that it's reasonable to just drop this type